zanata-war/src/main/java/org/zanata/webtrans/server/TranslationWorkspaceManagerImpl.java

```diff
@@ -172,26 +177,47 @@ protected String getSessionId() {
     }
 
     @Observer(ProjectUpdate.EVENT_NAME)
-    public void projectUpdate(@Observes ProjectUpdate payload) {
-        projectUpdate(payload.getProject());
+    // transaction has already been committed and marked as rolled back only for
+    // current thread. We have to open a new transaction to load any lazy
+    // properties (otherwise exception like javax.resource.ResourceException:
+    // IJ000460: Error checking for a transaction: Transactions are not active)
+    @Async
```

Comment:
Just the Work class will still give me exceptions. I think it will try to reuse previous transaction. Not sure is this is a JEE behavior or seam behavior.